include/flang/common/Fortran-features.h

```diff
@@ -9,9 +9,9 @@
 #ifndef FORTRAN_COMMON_FORTRAN_FEATURES_H_
 #define FORTRAN_COMMON_FORTRAN_FEATURES_H_
```

Comment:
Should these guards be also renamed to FLANG_COMMON?

Reply:
They should probably be changed to `LLVM_FLANG_COMMON_FORTRAN_FEATURES_H` to match clang, but I think that should be a separate PR with a separate discussion